otwcode/otwarchive
# AO3-5993 Remove OpenID tables (open_id_authentication_associations and open_id_authentication_nonces)

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-5993 

## Purpose

What does this PR do? As per Jira issue

## Testing Instructions

How can the Archive's QA team verify that this is working as you intended?

Run the migration.


## References


## Credit

What name and pronouns should we use to credit you in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)? salt, they/them
